source/locales/zh-TW.json

```diff
@@ -214,24 +214,24 @@
   "more": "更多 …",
   "The": "這",
   "site-media-index-title": "此文於下列媒體中被提及。",
-  "Catalan": "加泰隆尼亞文",
-  "German": "德文",
-  "English": "英文",
-  "Spanish": "西班牙文",
-  "French": "法文",
-  "Italian": "義大利文",
-  "Dutch": "荷蘭文",
-  "Polish": "波蘭文",
-  "Portuguese": "葡萄牙文",
-  "Russian": "俄文",
-  "Swedish": "瑞典文",
+  "Catalan": "Catalan",
```

Comment:
oh, my mistake, seem other language translate do use the translation of the language name. (let me update the zh-CN.json  too)